gettsim/taxes/zve.py

```diff
@@ -118,7 +121,6 @@ def kinderfreibetrag(tax_unit, params, kindergeld_params):
         kifreib_total = params["kifreib_s_exm"]
 
     diff_kifreib = nokfb_lower - (kifreib_total * child_num_kg)
```

Comment:
Lowerscore in `kifreib` and german word for `child_num_kg`

Reply:
> Lowerscore in `kifreib`

what exactly?

Reply:
Either `kindfreibetrag`, but if we abbreviate a word, then use underscore. So this would be `ki_freib`, which looks a bit ugly. I would just go with the whole word!

Reply:
In any case we should stay consistent across variations of the identifier. No need to change the entire old code right now, but I just added this to GEP 1:

  Across variations that include the same identifier, this identifier should not be
  changed, even if it leads to long variable names (e.g., `kinderfreibetrag`,
  `zu_verst_e_ohne_kinderfreibetrag`). This makes searching for that identifier easier
  and less error-prone.